### ISSUE TITLE
keep2share dl now ok, renaming completed file don't work

### ISSUE THREAD

**Issue by kveiro** [pyload/pyload#2575]
Hi, thanks for working plugin :+1:
But here small problem with name of file.

Once the download is complete, the file "original.file.name.rar.<b>chunk0</b>" is not renamed to "original.file.name.<b>rar</b>".
I have two files (chunks and chunk0) in download directory, so i renamed the file manually.
IOError 22 is wrong of course :)

Here is my log:

22.08.2016 15:36:02 DEBUG HOSTER Keep2ShareCc[47]: Plugin version: 0.31
85 22.08.2016 15:36:02 DEBUG HOSTER Keep2ShareCc[47]: Plugin status: testing
86 22.08.2016 15:36:02 WARNING HOSTER Keep2ShareCc[47]: Plugin may be unstable
87 22.08.2016 15:36:02 DEBUG Redirected import module.plugins.accounts.Keep2ShareCc -> userplugins.accounts.Keep2ShareCc
88 22.08.2016 15:36:02 DEBUG Redirected import module.plugins.internal.Account -> userplugins.internal.Account
89 22.08.2016 15:36:02 INFO HOSTER Keep2ShareCc[47]: Grabbing link info...
90 22.08.2016 15:36:03 DEBUG HOSTER Keep2ShareCc[47]: Link info: {'status': 2, 'hash': {}, 'name': 'Easy Context Menu 1.3 Portable.zip', 'url': u'http://k2s.cc/file/1fb5aacb18029', 'pattern': {'S': u'1.4 MB', 'ID': u'1fb5aacb18029', 'N': u'Easy Context Menu 1.3 Portable.zip'}, 'size': 1468006}
91 22.08.2016 15:36:03 DEBUG HOSTER Keep2ShareCc[47]: Previous link info: {}
92 22.08.2016 15:36:03 INFO HOSTER Keep2ShareCc[47]: Link name: Easy Context Menu 1.3 Portable.zip
93 22.08.2016 15:36:03 INFO HOSTER Keep2ShareCc[47]: Link size: 1.40 MiB (1468006 bytes)
94 22.08.2016 15:36:03 INFO HOSTER Keep2ShareCc[47]: Link status: online
95 22.08.2016 15:36:03 INFO HOSTER Keep2ShareCc[47]: Processing url: http://k2s.cc/file/1fb5aacb18029/Easy_Context_Menu_1.3_Portable.zip
96 22.08.2016 15:36:03 DEBUG HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={} | ref=False | just_header=False
97 22.08.2016 15:36:04 INFO HOSTER Keep2ShareCc[47]: Processing as free download...
98 22.08.2016 15:36:04 DEBUG HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'slow_id': u'0785ba38b16b1', 'yt0': ''} | ref=True | just_header=False
99 22.08.2016 15:36:05 DEBUG CAPTCHA Keep2ShareCc[47]: Captcha | LOAD URL http://k2s.cc/file/captcha.html?v=57baff9ee6675 | redirect=True | cookies=True | get={} | req= | decode=False | multipart=False | post={} | ref=False | just_header=False
100 22.08.2016 15:36:05 INFO CAPTCHA Keep2ShareCc[47]: Captcha | Using OCR to decrypt captcha...
101 22.08.2016 15:36:05 WARNING CAPTCHA Keep2ShareCc[47]: Captcha | No OCR result
102 22.08.2016 15:36:17 INFO CAPTCHA Keep2ShareCc[47]: Captcha | Captcha result: qejrdo
103 22.08.2016 15:36:17 DEBUG HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'CaptchaForm[code]': 'qejrdo', 'yt0': '', 'freeDownloadRequest': 1, 'uniqueId': u'0785ba38b16b1', 'free': 1} | ref=True | just_header=False
104 22.08.2016 15:36:17 INFO CAPTCHA Keep2ShareCc[47]: Captcha | Correct captcha | qejrdo
105 22.08.2016 15:36:17 DEBUG HOSTER Keep2ShareCc[47]: WAIT set to timestamp 1471873008.927000 | Previous waitUntil: 0.000000
106 22.08.2016 15:36:17 INFO HOSTER Keep2ShareCc[47]: Waiting 31 seconds...
107 22.08.2016 15:36:49 DEBUG HOSTER Keep2ShareCc[47]: LOAD URL http://k2s.cc/file/1fb5aacb18029 | redirect=True | cookies=True | get={} | req=None | decode=True | multipart=False | post={'free': '1', 'uniqueId': u'0785ba38b16b1'} | ref=True | just_header=False
108 22.08.2016 15:36:50 INFO HOSTER Keep2ShareCc[47]: Downloading file...
109 22.08.2016 15:36:50 DEBUG HOSTER Keep2ShareCc[47]: DOWNLOAD URL /file/url.html?file=0785ba38b16b1 | cookies=True | resume=None | get={} | disposition=True | chunks=None | post={} | ref=True
110 22.08.2016 15:36:53 DEBUG Content-Disposition: Easy Context Menu 1.3 Portable.zip filename*=UTF-8Easy%20Context%20Menu%201.3%20Portable.zip
111 22.08.2016 15:37:21 ERROR HOSTER Keep2ShareCc[47]:
112 22.08.2016 15:37:21 DEBUG ADDON ExternalScripts: No script found under folder download_processed
113 22.08.2016 15:37:21 DEBUG ADDON ExternalScripts: No script found under folder package_processed
114 22.08.2016 15:37:21 WARNING Download failed: Easy Context Menu 1.3 Portable.zip | IOError 22
115 22.08.2016 15:37:21 DEBUG ADDON ExternalScripts: No script found under folder download_failed
116 22.08.2016 15:37:21 INFO ADDON UnSkipOnFail: Looking for skipped duplicates of: Easy Context Menu 1.3 Portable.zip (pid:47)
117 22.08.2016 15:37:21 INFO ADDON UnSkipOnFail: No duplicates found 

Thank in advance !
